src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { TodoWithUser } from '../../types';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './TodoInfo.scss';
+
+interface Props {
+  todo: TodoWithUser;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    user,
+    completed = false,
+    id,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user
+        ? <UserInfo user={user} />
+        : <p> No user found</p>}
```

Comment:
```suggestion
      {user
        ? <UserInfo user={user} />
        : <p> No user found</p>
      }
```

Reply:
Not fixed 😢

Reply:
Ok, let's keep it then
Just one thing (optional to fix) - I would rather handle an empty user with the component to avoid this external check every time it is used.